package.json

```diff
@@ -3,48 +3,14 @@
   "version": "0.1.2",
   "private": true,
   "jest": {
-    "setupEnvScriptFile": "./node_modules/react-native/jestSupport/env.js",
-    "haste": {
-      "defaultPlatform": "ios",
-      "platforms": [
-        "ios",
-        "android"
-      ],
-      "providesModuleNodeModules": [
-        "react-native"
-      ]
+    "preset": "jest-react-native",
+    "moduleNameMapper": {
+      "\\.\/datepicker$": "<rootDir>/node_modules/tcomb-form-native/lib/templates/bootstrap/datepicker.ios.js"
```

Comment:
this one is curious. Jest doesn't seem to resolve relative platform paths properly but I'm not sure I'm interested in supporting this. For now, this mapping works.